File: Python/Lab_2/Lab_2.py
from lib import *

name_out = "Files/" + input("Enter a file name: ")

output_file = create_file(name_out)
input_file(output_file)
file_output(name_out)
the_nearest_case(name_out)

name_in = "Files/" + input("Enter a file name: ")
free_time(name_out, name_in)
b_file_output(name_in)

File: C++/Lab_2/Lab_2.cpp
#include "func.h"

int main()
{
	ofstream file_out;
	string name_out = create_file(file_out);
	input_file(file_out);
	file_output(name_out);
	the_nearest_case(name_out);
	file_out.close();

	string name_in = free_time(name_out);
	b_file_output(name_in);

	return 0;
}